grunt/tasks/misc.js

```diff
@@ -96,11 +97,11 @@ module.exports = grunt => {
     'copy:deploy_favicon',
     'includereplace:prod_index',
     'includereplace:prod_auth',
+    'includereplace:prod_auth_new',
     'clean:deploy_app',
     'uglify:deploy',
     'concat:deploy',
     'clean:deploy_script',
-    'raygun:prod',
```

Comment:
If we actually don't need it anymore.. let's remove the whole task to clean up things

Reply:
Done